packages/fluentui/CHANGELOG.md

```diff
@@ -147,6 +147,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add new sample images for use in Avatar examples @behowell ([#16129](https://github.com/microsoft/fluentui/pull/16129))
 - Update `Form` examples to remove `control` prop and delete shorthand examples @assuncaocharles ([#16467](https://github.com/microsoft/fluentui/pull/16467))
 
+### Deprecated
```

Comment:
```suggestion
### Deprecations
```

??

Reply:
Let's also update the title in this file to reflect sections order

Reply:
![image](https://user-images.githubusercontent.com/14183168/104617413-b316f080-568b-11eb-82ab-d0bd289ac896.png)

Reply:
I have no preference here tbh, both get the message across equally IMO